docs/src/javascript/README.md

```diff
@@ -16,8 +16,8 @@ npm install vectordb
 
 ```javascript
 const lancedb = require('vectordb');
-const db = await lancedb.connect('data/sample-lancedb');
-const table = await db.createTable("my_table", 
+const db = lancedb.connect('<PATH_TO_LANCEDB_DATASET>');
```

Comment:
```suggestion
const db = lancedb.connect('data/sample-lancedb');
```
My fault for the oversight with the node readme, want to keep this consistent with other uri's in docs